content/Rustup-1.28.2.md

```diff
@@ -0,0 +1,84 @@
++++
+path = "2025/04/25/Rustup-1.28.2"
+title = "Announcing rustup 1.28.2"
+authors = ["The Rustup Team"]
+aliases = ["2025/04/25/Rustup-1.28.2.html"]
++++
+
+The rustup team is happy to announce the release of rustup version 1.28.2.
+[Rustup][install] is the recommended tool to install [Rust][rust], a programming language that
+empowers everyone to build reliable and efficient software.
+
+## What's new in rustup 1.28.2
+
+The headlines of this release are:
+
+- The cURL download backend and the native-tls TLS backend are now officially deprecated and
+  a warning will start to show up when they are used. [pr#4277]
+
+  - While rustup predates reqwest and rustls, the rustup team has long wanted to standardize on
+    an HTTP + TLS stack in Rust, which should increase security, potentially improve performance, and
+    simplify maintenance of the project.
+    With the default download backend already switched to reqwest since [2019][pr#1660], the team
+    thinks it is time to focus maintenance on the Rust-based stack.
+
+  - The rustup team encourages everyone to switch to the reqwest backend with rustls, and would love
```

Comment:
Presumably there's no actual work involved here in this encouragement, right? This is just the default with the new release? Can we rephrase this to make that clear?